cmd/single-file.go

```diff
@@ -256,14 +257,40 @@ func (c *SingleFileCmd) Run() error {
 
 	mlog.Verbose("generating output artifact %s", outfile)
 
-	// run gen script
-	cmd := exec.Command(
-		"/usr/bin/single-file-artifact-gen",
-		"-n", c.ArtifactName,
-		"-t", c.DeviceType,
+	deviceTypes := strings.Split(c.DeviceType, ",")
```

Comment:
In spirit, this looks correct, but I'll leave for someone else to evaluate whether this is correct place to do it.